sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/DataCollection/DocumentHelper.cs

```diff
@@ -191,7 +191,13 @@ internal static Request ConvertToRequestDocument(Activity activity)
                 }
                 else if (tag.Key == SemanticConventions.AttributeServerPort)
                 {
-                    serverPort = tag.Value.ToString()!;
+                    object port80 = 80;
+                    object port443 = 443;
+
+                    if (tag.Value != port80 && tag.Value != port443)
```

Comment:
Do we have test coverage for this feature?

Reply:
added tests. please re-review.